### PR TITLE
Format ctrl code

### PR DESCRIPTION
**Pull request by mPokornyETM**
Format ctrl code to be more consistent and aply ETM ctrl code rules

### CODE REVIEW

File: WinCCOA_QualityChecks/scripts/libs/scriptEditor/ctrlPPCheck_ext.ctl [hunk 1]
```diff
@@ -54,47 +54,52 @@ void ctrlPPCheck()
...
+  if (isFunctionDefined("seGetFileName"))
+    path = seGetFileName();
 
   // store path;
   map["path"] = path;
 
   bool tmpFileUsed = FALSE;
-  if ( path == "" ) // panel or version < 3.16
+
+  if (path == "")   // panel or version < 3.16
```
Comment (dhoegerlETM) [inline WinCCOA_QualityChecks/scripts/libs/scriptEditor/ctrlPPCheck_ext.ctl:65]: Consider refactoring: better check path.isEmpty()

**Review by dhoegerlETM — APPROVED** [review]
Only formatting changes, except for one "else-if" instead single "if", which should be OK.

File: WinCCOA_QualityChecks/scripts/libs/scriptEditor/ctrlPPCheck_ext.ctl (hunk 1 above)
Comment (mPokornyETM) [reply WinCCOA_QualityChecks/scripts/libs/scriptEditor/ctrlPPCheck_ext.ctl:65]: Do we want it?, It will never works with WinCC OA < 3.18

Comment (dhoegerlETM) [reply WinCCOA_QualityChecks/scripts/libs/scriptEditor/ctrlPPCheck_ext.ctl:65]: Not necessary - was only a comment. But you are right for support-versions.

**Pull request merged**